### PR TITLE
CAD-1230:  integrate txgen with shelley3pools

### PR DESCRIPTION
**Pull request by deepfire**

### CODE REVIEW

File: benchmarks/shelley3pools/configuration/configuration-node-1.yaml [hunk 1]
```diff
@@ -114,19 +114,19 @@ traceForwardTo:
 # be specified in the defaults below or overidden on a per-scribe basis here.
 setupScribes:
   - scKind: FileSK
-    scName: "logs/node1.json"
+    scName: "logs/node-1.json"
```
Comment (CodiePP) [inline benchmarks/shelley3pools/configuration/configuration-node-1.yaml]: please, don't change these names. The analyse scripts and the Readme access the logs via these defined names.

**Review by CodiePP — CHANGES_REQUESTED** [review]
lgtm, just don't change the names of the logs

File: benchmarks/shelley3pools/configuration/configuration-node-1.yaml (hunk 1 above)
Comment (deepfire) [reply benchmarks/shelley3pools/configuration/configuration-node-1.yaml]: Done.

**Review by CodiePP — APPROVED** [review]
now these changes are fine and minimal.

**Pull request merged**